### PR TITLE
Added tabs component

### PR DESCRIPTION
**Pull request by Anton-serdiukov**

### CODE REVIEW

File: src/components/Tabs.vue [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    display: flex;
+    flex-direction: column;
+
+    width: 100%;
+    height: auto;
+    box-shadow: 0 1px 0 0 @color-gray-300;
+
+    .content {
+      width: 100%;
+      margin-bottom: 20px;
```
Comment (HighQualityCode) [inline src/components/Tabs.vue]: For the consistence of the application, please add the spacing variables into vars.less file and use the variables here.

File: src/components/Tabs.vue [hunk 2]
```diff
@@ -0,0 +1,142 @@
+<!--
+  Button with an optional icon and different styles
+
+  Usage:
+
+    <Tabs
+      :tabs="{
+          defaultTab: 0, <--- ITEM INDEX (Number)
```
Comment (Igonato) [inline src/components/Tabs.vue]: This should be a separate property, maybe change active to a property? Then you can just pass the array here

File: src/components/Tabs.vue [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            {
+              text: 'TAB 3',  (Number)
+              hidden: true,   (Function)
+              disabled: true  (Boolean)
+            }
+          ]
+        }"
+    >
+      <template>
+          SOME ADDITIONAL COMPONENT FOR
```
Comment (Igonato) [inline src/components/Tabs.vue]: ??? There aren't any design requirements to be able to pass anything as a slot. Please, remove

File: src/components/Tabs.vue [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  Button with an optional icon and different styles
+
+  Usage:
+
+    <Tabs
+      :tabs="{
+          defaultTab: 0, <--- ITEM INDEX (Number)
+          items: [
+            {
+              text: 'TAB 1',  (Number)
```
Comment (Igonato) [inline src/components/Tabs.vue]: Number? Please fix these type annotations

File: src/components/Tabs.vue [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+        }"
+    >
+      <template>
+          SOME ADDITIONAL COMPONENT FOR
+      </template>
+    </Tabs>
+
+    Events:
+
+    @tab:click - Return tab text in snake case
```
Comment (Igonato) [inline src/components/Tabs.vue]: I think it should just return the tab item from items array, And we can any data we want to it

File: src/components/Tabs.vue [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+  <div class="tabs-container">
+    <div class="content">
+      <slot></slot>
+    </div>
+
+    <div class="tabs-row">
+      <template v-for="tab in tabs.items">
+        <div
+          v-if="!tab.hidden"
+          :disabled="tab.disabled"
```
Comment (Igonato) [inline src/components/Tabs.vue]: Have you tested this? Doesn't seem to work for me

File: src/components/Tabs.vue [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: "Tabs",
+    props: {
+      defaultTab: {
```
Comment (Igonato) [inline src/components/Tabs.vue]: I see you've already made it into a separate property. Please update the document at the top. And I would still merge it with `active`

File: src/components/Tabs.vue [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+      },
+
+      tabs: {
+        required: true,
+        type: Object
+      }
+    },
+    data() {
+      return {
+        size: 'middle',
```
Comment (Igonato) [inline src/components/Tabs.vue]: I see this is added as a class to the code, but nothing in the styles. Can this be removed?

File: src/components/Tabs.vue [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+        active: this.tabs.items[this.tabs.defaultTab || 0].text
+      }
+    },
+    mounted() {},
```
Comment (Igonato) [inline src/components/Tabs.vue]: Remove

**Pull request merged**